__tests__/app.test.js

```diff
@@ -211,4 +211,13 @@ describe("NC_NEWS", () => {
             });
         });
     });
+
+    describe('/api/comments/:comment_id', () => {
+        // task 9
+        test("DELETE 204: delete the given comment by comment_id", () => {
+            return request(app)
+            .delete('/api/comments/4')
+            .expect(204)
+        })
```

Comment:
Please add two more tests here where the ID is invalid and one where it's valid but doesn't exist.